### PR TITLE
Add image to the book details

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

This will add the new image for the books using flex. Also I added a little fixes in the indentation, and the usage of the classes. Some classes were useless or needed to be updated.

## Screenshots 
Screenshot from 2020-02-28 18-47-40

## Cards

https://trello.com/c/wHsGRZ43/27-detalle-de-libro-ii-imagen

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,24 +9,28 @@
...
-    </div> 
-  </div>
+  <div class = book-cover-page>
+    <img class = book-img src="images/book-cover.png" alt="Book Cover">
+    <div class="book-info">
+      <header class="book-header"> 
+        <h1 class="book-title">Título del libro</h1>
+        <p class="book-genre">(género)</p>
+      </header>
+          <div class="book-content-row">
```
Comment (CrisVega08) [inline index.html]: check indentation

File: index.html [hunk 2]
```diff
@@ -9,24 +9,28 @@
...
-    <div class="book-content-row">
-      <h3 class="default-text category">Editorial:</h3>
-      <p class="default-text">Nombre de la editorial</p>
-    </div>
-    <div class="book-content-row"> 
-      <h3 class="default-text category">Año de publicación:</h3>
-      <p class="default-text">Año de publicación</p>
-    </div> 
-  </div>
+  <div class = book-cover-page>
```
Comment (CrisVega08) [inline index.html]:
```suggestion
  <div class=book-cover-page>
```

File: index.html [hunk 3]
```diff
@@ -9,24 +9,28 @@
...
+    <img class = book-img src="images/book-cover.png" alt="Book Cover">
```
Comment (CrisVega08) [inline index.html]:
```suggestion
    <img class="book-img" src="images/book-cover.png" alt="Book-Cover">
```

File: scss/detail.scss [hunk 4]
```diff
@@ -4,12 +4,14 @@
...
+  margin: 15px;
+}
+
+.book-cover-page {
   display: flex;
-  flex-direction: column;
-  height: 125px;
-  justify-content: space-between;
-  margin-left: 10px;
+  flex-direction: row;
```
Comment (CrisVega08) [inline scss/detail.scss]: This is default value so you don't need it

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (eledwinn) [inline index.html]:
```suggestion
  <div class="book-cover-page">
```

File: scss/detail.scss [hunk 5]
```diff
@@ -4,12 +4,14 @@
   font-family: Roboto
```
Comment (eledwinn) [inline scss/detail.scss]: Please add `;` at end of line.

**Review by eledwinn — CHANGES_REQUESTED** [review]

**Review by eledwinn — APPROVED** [review]

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**